selfdrive/test/process_replay/process_replay.py

```diff
@@ -186,7 +187,13 @@ def radar_rcv_callback(msg, CP, cfg, fsm):
 def calibration_rcv_callback(msg, CP, cfg, fsm):
   # calibrationd publishes 1 calibrationData every 5 cameraOdometry packets.
   # should_recv always true to increment frame
-  recv_socks = ["liveCalibration"] if (fsm.frame + 1) % 5 == 0 else []
+  recv_socks = []
+  if msg.which() == 'carState':
+    if not hasattr(fsm, 'cs_counter'):
+      fsm.cs_counter = 0
+    if (fsm.cs_counter % 25) == 0:
+      recv_socks = ["liveCalibration"]
+    fsm.cs_counter += 1
   return recv_socks, True
```

Comment:
I think if you only return True for should_recv when it's a carState msg, you can just use fsm.frame instead of adding this counter.

Something like:
```
def calibration_rcv_callback(msg, CP, cfg, fsm):
  recv_socks = ["liveCalibration"] if (fsm.frame + 1) % 25 == 0 else []
  return recv_socks, msg.which() == 'carState'
```

Reply:
ah great, thanks!